js/src/dapps/localtx/Transaction/transaction.js

```diff
@@ -256,23 +259,29 @@ export class LocalTransaction extends BaseTransaction {
 
     const newTransaction = {
       from: transaction.from,
-      to: transaction.to,
```

Comment:
So the issue was that the `to` field was set, and thus formatted to `hex`: `'' => '0x'`.
I guess the API should be modified in order to take this empty `to` case into account.